app/controllers/api/v2/hosts_controller.rb

```diff
@@ -203,6 +203,21 @@ def facts
         render_message(e.to_s, :status => :unprocessable_entity)
       end
 
+      api :POST, "/hosts/:id/rebuild_config", N_("Rebuild orchestration config (TFTP, DNS and DHCP)")
+      param :id, :identifier_dottable, :required => true
+      def rebuild_config
+        result = @host.recreate_config
+        failures = []
+        result.each_pair do |key, value|
+          failures << key.to_s unless value
```

Comment:
Ditto what I said for the UI, but is `key` here the pretty name?  If so, call `_(key)` so we get the translated version.